src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CaseRepository.java

```diff
@@ -64,5 +64,7 @@ void saveCountyCourtJudgment(
     void saveCaseEvent(String authorisation, Claim claim, CaseEvent caseEvent);
 
     Claim saveClaimDocuments(String authorisation, Long claimId, ClaimDocumentCollection claimDocumentCollection);
+
+    Claim linkLetterHolder(Long claimId, String letterHolderId, String authorisation);
```

Comment:
Authorisation is not required. Remove it please

Reply:
good pick